docs/guides/mysql/pitr/restic/archiver.md

```diff
@@ -0,0 +1,459 @@
+---
+title: Continuous Archiving and Point-in-time Recovery Using Restic Driver
+menu:
+  docs_{{ .version }}:
+    identifier: restic
+    name: Restic
+    parent: pitr-mysql
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# KubeDB MySQL - Continuous Archiving and Point-in-time Recovery Using Restic Driver
+
+Here, we will demonstrate how to use KubeDB to provision a MySQL database for continuous archiving and point-in-time restoration.
```

Comment:
"Here, we will demonstrate how to use KubeDB to provision a MySQL database with continuous archiving capabilities, also show point-in-time restoration."